class-tgm-plugin-activation.php

```diff
@@ -517,7 +517,12 @@ protected function do_plugin_install() {
 					$api = plugins_api( 'plugin_information', array( 'slug' => $plugin['slug'], 'fields' => array( 'sections' => false ) ) );
 
 					if ( is_wp_error( $api ) ) {
-						wp_die( $this->strings['oops'] . var_dump( $api ) );
+						if ( WP_DEBUG === true ) {
```

Comment:
No need, WP defines it as `false` if not set: https://core.trac.wordpress.org/browser/trunk/src/wp-includes/default-constants.php#L59